README.md

```diff
@@ -40,3 +41,39 @@ pip install pipenv
 pipenv install --dev
 pipenv run uvicorn example.app:app
 ```
+
+## Usage without minor version
+```python
+from fastapi import FastAPI
+from fastapi_versioning import VersionedFastAPI, version
+
+app = FastAPI(title='My App')
+
+@app.get('/greet')
+@version(1)
+def greet():
+  return 'Hello'
+
+@app.get('/greet')
+@version(2)
+def greet():
+  return 'Hi'
+
+app = VersionedFastAPI(app,
+    version_format='{major}',
+    prefix_format='/v{major}')
+```
+
+this will generate two endpoints:
+```
+/v1/greet
+/v1/greet
```

Comment:
/v2/greet

Reply:
'/v2/greet' at line 70 indeed.
Just let me know if you update directly or want me to do it.